strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -131,10 +139,10 @@ protected String createMessage(HttpServletRequest request,
         return result;
     }
 
-    @Bean
-    HeaderMappingFilter headerMappingFilter()
+    @Override
+    protected RequestMappingHandlerMapping createRequestMappingHandlerMapping()
```

Comment:
The name sounds a bit weird... Wouldn't `RequestMappingHandler` be a better name for this?